src/operator/utils.ts

```diff
@@ -7,6 +7,12 @@ import {
 
 // Types
 import {OperatorOrgLimits, OperatorRegions} from 'src/types'
+import {IdentityUser} from 'src/client/unityRoutes'
+
+type OperatorRole = IdentityUser['operatorRole'] | null
```

Comment:
do we need the ` | null` here? `OperatorRole` is defined with a `?` and the function checks the only two values possible for `operatorRule`